utopia-vscode-common/package-lock.json

```diff
@@ -424,6 +424,16 @@
       "dev": true,
       "optional": true
     },
+    "bindings": {
```

Comment:
Interesting that this file changed. Please can you revert it from this PR since it's unrelated.